apps/expo-go/ios/Podfile.lock

```diff
@@ -2021,7 +2027,7 @@ SPEC CHECKSUMS:
   GoogleDataTransport: 54dee9d48d14580407f8f5fbf2f496e92437a2f2
   GoogleMaps: a146f275ee429d14822178c7a841c03366ec92a1
   GoogleUtilities: 13e2c67ede716b8741c7989e26893d151b2b2084
-  hermes-engine: 0933c22606de894dccc3248ae93129951a6e8e6b
```

Comment:
it bumped hermes for me after removing Pods – should I do something else to avoid it?

Reply:
Ok, let's ignore it then. Maybe it was just not correct in Expo Go 🤷